### DISCUSSION TITLE
Graphic flag showing that a field has declared conditions

### DISCUSSION THREAD

**Discussion by uebmaster** [directus/directus#10170] [Feature Requests]
A graphic element could be added to the data model edition interface to be able to know at a glance that a field has declared conditions, without having to open the field detail and open the conditions tab.

**Comment by joselcvarela** [comment]
I think it would be nicer if we have a place we could view a resume of the field like in a big tooltip:
|||
|---|---|
|Type|Integer|
|Relationship| M2M: `junction` <-> `related`|
|Interface|M2M: {Display Template}|
|Display|Related Values: {Display Template}|
|Conditions| Defined [Open]

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
